### ISSUE TITLE
Quokka (1.0.222) Plugin under Intellij IDEA gets null pointer exception

### ISSUE THREAD

**Issue by dimylik** [wallabyjs/quokka#463]
### Sample code

```javascript
// paste the code you are running here
import slide_out from 'devextreme/ui/slide_out';

const existingParamList = [1,2,3];
const mnemonic=5;
const updatedParam = [...existingParamList, mnemonic];
console.log(updatedParam);
console.log(updatedParam.join(','));

function getSplit(b) {
    return b && b.toString() && [b].join(',').split(',') || [];
}

let a = undefined;

a = [1,2,3];
console.log('xxx', getSplit(a))

a = '1,2,3';
console.log('xxx', getSplit(a))

a = '';
console.log('xxx', getSplit(a))

a = [];
console.log('xxx', getSplit(a))

console.log('xxx', getSplit(undefined))
console.log('xxx', getSplit(null))

console.log('array is', [].toString() ? 'not empty' : 'empty');

console.log('empty array length is', [].join(',').length);

```

### Sample repository link

*If the issue can not be reproduced just using the quokka file above 
(for example because it requires/imports some files from your project), 
please create a small repository where the issue can be reproduced.* 

### Quokka.js Console Output

```
// paste your quokka console output here
Following exceptions happened soon after this one, most probably they are induced.


2020-02-26T10:06:48.273
java.lang.NullPointerException
	at com.quokka.javascript.session.a.selectionChanged(Unknown Source)
	at com.intellij.openapi.editor.impl.SelectionModelImpl.broadcastSelectionEvent(SelectionModelImpl.java:130)
	at com.intellij.openapi.editor.impl.SelectionModelImpl.fireSelectionChanged(SelectionModelImpl.java:124)
	at com.intellij.openapi.editor.impl.CaretImpl.lambda$doSetSelection$6(CaretImpl.java:1196)
	at com.intellij.openapi.editor.impl.CaretModelImpl.doWithCaretMerging(CaretModelImpl.java:453)
	at com.intellij.openapi.editor.impl.CaretImpl.doSetSelection(CaretImpl.java:1113)
	at com.intellij.openapi.editor.impl.CaretImpl.setSelection(CaretImpl.java:1103)
	at com.intellij.openapi.editor.impl.CaretImpl.setSelection(CaretImpl.java:1095)
	at com.intellij.openapi.editor.impl.SelectionModelImpl.setSelection(SelectionModelImpl.java:101)
	at com.intellij.openapi.editor.impl.EditorImpl.setSelectionAndBlockActions(EditorImpl.java:2664)
	at com.intellij.openapi.editor.impl.EditorImpl.processMouseDragged(EditorImpl.java:2572)
	at com.intellij.openapi.editor.impl.EditorImpl.lambda$runMouseDraggedCommand$14(EditorImpl.java:2422)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:220)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:178)
	at com.intellij.openapi.editor.impl.EditorImpl.runMouseDraggedCommand(EditorImpl.java:2422)
	at com.intellij.openapi.editor.impl.EditorImpl.access$11600(EditorImpl.java:123)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseMotionListener.lambda$mouseDragged$0(EditorImpl.java:4174)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:195)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseMotionListener.mouseDragged(EditorImpl.java:4174)
	at java.awt.Component.processMouseMotionEvent(Component.java:6598)
	at javax.swing.JComponent.processMouseMotionEvent(JComponent.java:3343)
	at java.awt.Component.processEvent(Component.java:6319)
	at java.awt.Container.processEvent(Container.java:2239)
	at java.awt.Component.dispatchEventImpl(Component.java:4899)
	at java.awt.Container.dispatchEventImpl(Container.java:2297)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4904)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4552)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4476)
	at java.awt.Container.dispatchEventImpl(Container.java:2283)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:766)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:739)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:736)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:741)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:686)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:385)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

2020-02-26T10:06:48.605
java.lang.NullPointerException
	at com.quokka.javascript.session.a.selectionChanged(Unknown Source)
	at com.intellij.openapi.editor.impl.SelectionModelImpl.broadcastSelectionEvent(SelectionModelImpl.java:130)
	at com.intellij.openapi.editor.impl.SelectionModelImpl.fireSelectionChanged(SelectionModelImpl.java:124)
	at com.intellij.openapi.editor.impl.CaretImpl.lambda$doSetSelection$6(CaretImpl.java:1196)
	at com.intellij.openapi.editor.impl.CaretModelImpl.doWithCaretMerging(CaretModelImpl.java:453)
	at com.intellij.openapi.editor.impl.CaretImpl.doSetSelection(CaretImpl.java:1113)
	at com.intellij.openapi.editor.impl.CaretImpl.setSelection(CaretImpl.java:1103)
	at com.intellij.openapi.editor.impl.CaretImpl.setSelection(CaretImpl.java:1095)
	at com.intellij.openapi.editor.impl.SelectionModelImpl.setSelection(SelectionModelImpl.java:101)
	at com.intellij.openapi.editor.impl.EditorImpl.setSelectionAndBlockActions(EditorImpl.java:2664)
	at com.intellij.openapi.editor.impl.EditorImpl.processMouseDragged(EditorImpl.java:2572)
	at com.intellij.openapi.editor.impl.EditorImpl.lambda$runMouseDraggedCommand$14(EditorImpl.java:2422)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:220)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:178)
	at com.intellij.openapi.editor.impl.EditorImpl.runMouseDraggedCommand(EditorImpl.java:2422)
	at com.intellij.openapi.editor.impl.EditorImpl.access$11600(EditorImpl.java:123)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseMotionListener.lambda$mouseDragged$0(EditorImpl.java:4174)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:195)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseMotionListener.mouseDragged(EditorImpl.java:4174)
	at java.awt.Component.processMouseMotionEvent(Component.java:6598)
	at javax.swing.JComponent.processMouseMotionEvent(JComponent.java:3343)
	at java.awt.Component.processEvent(Component.java:6319)
	at java.awt.Container.processEvent(Container.java:2239)
	at java.awt.Component.dispatchEventImpl(Component.java:4899)
	at java.awt.Container.dispatchEventImpl(Container.java:2297)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4904)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4552)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4476)
	at java.awt.Container.dispatchEventImpl(Container.java:2283)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4721)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:766)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
	at java.awt.EventQueue$4.run(EventQueue.java:739)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:736)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:741)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:686)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:385)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

### Code editor version

Intellij IDEA 2019.1

### OS name and version

Windows

**Comment by dimylik** [comment]
// code I have tested under scratch.ts

import slide_out from 'devextreme/ui/slide_out';

const existingParamList = [1,2,3];
const mnemonic=5;
const updatedParam = [...existingParamList, mnemonic];
console.log(updatedParam);
console.log(updatedParam.join(','));

function getSplit(b) {
    return b && b.toString() && [b].join(',').split(',') || [];
}

let a = undefined;

a = [1,2,3];
console.log('xxx', getSplit(a))

a = '1,2,3';
console.log('xxx', getSplit(a))

a = '';
console.log('xxx', getSplit(a))

a = [];
console.log('xxx', getSplit(a))

console.log('xxx', getSplit(undefined))
console.log('xxx', getSplit(null))


console.log('array is', [].toString() ? 'not empty' : 'empty');

console.log('empty array length is', [].join(',').length);

**Comment by NikGovorov** [comment]
Thanks for reporting the problem. We have released a new version of Quokka for JetBrains IDEs that hopefully fixes the problem (v1.0.223).
